client/payment-details/dispute-details/dispute-steps.tsx

```diff
@@ -0,0 +1,202 @@
+	let emailLink;
+	if ( customer?.email ) {
+		const chargeDate = dateI18n(
+			'Y-m-d',
+			moment( chargeCreated * 1000 ).toISOString()
+		);
+		const disputeDate = dateI18n(
+			'Y-m-d',
+			moment( dispute.created * 1000 ).toISOString()
+		);
+		const emailSubject = sprintf(
+			// Translators: %1$s is the store name, %2$s is the charge date.
+			__(
+				`Problem with your purchase from %1$s on %2$s?`,
+				'woocommerce-payments'
+			),
+			wcpaySettings.storeName,
+			chargeDate
+		);
+		const customerName = customer?.name || '';
+		const emailBody = sprintf(
+			// Translators: %1$s is the customer name, %2$s is the dispute date, %3$s is the dispute amount with currency-code e.g. $15 USD, %4$s is the charge date.
+			__(
+				`Hello %1$s\n\n` +
+					`We noticed that on %2$s, you disputed a %3$s charge on %4$s. We wanted to contact you to make sure everything was all right with your purchase and see if there's anything else we can do to resolve any problems you might have had.\n\n` +
+					`Alternatively, if the dispute was a mistake, you can easily withdraw it by calling the number on the back of your card. Thank you so much - we appreciate your business and look forward to working with you.`,
+				'woocommerce-payments'
+			),
+			customerName,
+			disputeDate,
+			formatExplicitCurrency( dispute.amount, dispute.currency ),
+			chargeDate
+		);
+		emailLink = `mailto:${ customer.email }?subject=${ encodeURIComponent(
+			emailSubject
+		) }&body=${ encodeURIComponent( emailBody ) }`;
+	}
+
+	const respondByDate = dispute.evidence_details?.due_by
+		? dateI18n(
+				'M j, Y, g:ia',
+				moment( dispute.evidence_details?.due_by * 1000 ).toISOString()
+		  )
+		: '–';
+
+	return (
+		<div className="dispute-steps">
+			<div className="dispute-steps__header">
+				{ __( 'Steps to resolve:', 'woocommerce-payments' ) }
+			</div>
+			<ol className="dispute-steps__steps">
+				<li>
+					{ customer?.email
+						? createInterpolateElement(
+								__(
+									'<a>Email the customer</a> to address their concerns.',
+									'woocommerce-payments'
+								),
+								{
+									a: (
+										// eslint-disable-next-line jsx-a11y/anchor-has-content
+										<a
+											target="_blank"
+											rel="noopener noreferrer"
+											href={ emailLink }
+										/>
+									),
+								}
+						  )
+						: __(
+								'Email the customer to address their concerns.',
+								'woocommerce-payments'
+						  ) }
+				</li>
+				<li>
+					{ createInterpolateElement(
+						__(
+							'Provide <a>guidance on dispute withdrawal</a> if the customer agrees.',
+							'woocommerce-payments'
+						),
+						{
+							a: (
+								<ExternalLink href="https://woocommerce.com/document/woopayments/fraud-and-disputes/managing-disputes/#withdrawals" />
+							),
+						}
+					) }
+				</li>
+				<li>
+					{ createInterpolateElement(
+						__(
+							'Challenge <challengeicon/> or accept <accepticon/> the dispute by <disputeduedate/>.',
```

Comment:
I found these inline icon tooltips a bit weird, they break the flow of the sentence. It's functioning well though and super clear and usable so not a blocker. 

Maybe that info could be in a single help popup, nearer the action buttons. Thinking out loud – @nikkivias  is our UX expert here.